[2025-03-31T13:19:25.661-0500] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-31T13:19:25.666-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exec_mult_tasks.taskG manual__2025-03-31T18:18:56.165989+00:00 [queued]>
[2025-03-31T13:19:25.670-0500] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exec_mult_tasks.taskG manual__2025-03-31T18:18:56.165989+00:00 [queued]>
[2025-03-31T13:19:25.670-0500] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-03-31T13:19:25.828-0500] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): taskG> on 2025-03-31 18:18:56.165989+00:00
[2025-03-31T13:19:25.830-0500] {standard_task_runner.py:72} INFO - Started process 425485 to run task
[2025-03-31T13:19:25.831-0500] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'exec_mult_tasks', 'taskG', 'manual__2025-03-31T18:18:56.165989+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/exec_multiple_tasks.py', '--cfg-path', '/tmp/tmpm6zn5yw7']
[2025-03-31T13:19:25.832-0500] {standard_task_runner.py:105} INFO - Job 46: Subtask taskG
[2025-03-31T13:19:25.969-0500] {task_command.py:467} INFO - Running <TaskInstance: exec_mult_tasks.taskG manual__2025-03-31T18:18:56.165989+00:00 [running]> on host royal-07.cs.wisc.edu
[2025-03-31T13:19:26.218-0500] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='cloud.user' AIRFLOW_CTX_DAG_ID='exec_mult_tasks' AIRFLOW_CTX_TASK_ID='taskG' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T18:18:56.165989+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T18:18:56.165989+00:00'
[2025-03-31T13:19:26.219-0500] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-31T13:19:26.227-0500] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-31T13:19:26.227-0500] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', 'tmp8a_wayxx.sh']
[2025-03-31T13:19:26.230-0500] {subprocess.py:99} INFO - Output:
[2025-03-31T13:19:26.231-0500] {subprocess.py:106} INFO - TASK G completed!
[2025-03-31T13:19:26.231-0500] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-31T13:19:26.343-0500] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-31T13:19:26.343-0500] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=exec_mult_tasks, task_id=taskG, run_id=manual__2025-03-31T18:18:56.165989+00:00, execution_date=20250331T181856, start_date=20250331T181925, end_date=20250331T181926
[2025-03-31T13:19:26.646-0500] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-31T13:19:26.651-0500] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-31T13:19:26.652-0500] {local_task_job_runner.py:245} INFO - ::endgroup::
